src/main/resources/static/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>WATCHTUBE</title>
+    <meta charset="UTF-8">
+    <link href="style.css" type="text/css" rel="stylesheet"/>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script src="https://unpkg.com/react@15/dist/react.js"></script>
+    <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.23/browser.min.js"></script>
+    <script src="userRequestFunctions.js" type="text/babel"></script>
+</head>
+<body>
+
+<div id="header">
+    <h1>
+        <a href="index.html">WATCHTUBE</a>
+    </h1>
+</div>
+
+<div id="signUpForm" class="forms">
```

Comment:
&#x1F34E; **Issue**
Could you generate "Forms" and the like through React ? As I understand the idea behind it - your application consists of components. You have your entry-point tag (<div id="app"></div> or something) and then you populate it by creating components.